app/src/main/java/org/schabi/newpipe/local/feed/FeedFragment.kt

```diff
@@ -101,6 +100,7 @@ class FeedFragment : BaseStateFragment<FeedState>() {
 
     private lateinit var groupAdapter: GroupieAdapter
     @State @JvmField var showPlayedItems: Boolean = true
+    @State @JvmField var showPartiallyPlayedItems: Boolean = true
     @State @JvmField var showFutureItems: Boolean = true
```

Comment:
Remove these three fields and just call the view model functions when building the `checkedDialogItems` boolean array below.